sig-scalability/slos/network_programming_latency.md

```diff
@@ -0,0 +1,89 @@
+## Network programming latency SLIs/SLOs details
+
+### User stories
+- As a user of vanilla Kubernetes, I want some guarantee how quickly new backends
+of my service will be targets of in-cluster load-balancing
+- As a user of vanilla Kubernetes, I want some guarantee how quickly deleted
+(or unhealthy) backends of my service will be removed from in-cluster
+load-balancing
+- As a user of vanilla Kubernetes, I want some guarantee how quickly changes
+to service specification (including creation) will be reflected in in-cluster
+load-balancing
+
+### Other notes
+- We are consciously focusing on in-cluster load-balancing for the purpose of
+this SLI, as external load-balancing is clearly provider specific (which makes
+it hard to set the SLO for it).
+- However, in the future it should be possible to formulate the SLI for external
+load-balancing in pretty much the same way for consistency.
+- The SLI measuring end-to-end time from pod creation was also considered,
```

Comment:
from -> for

Reply:
That's actually correct: from pod creation to when it's available via in-cluster lb.

Reply:
Ok, I misunderstood that.